apache/superset
# fix(plugin-chart-handlebars): fix overflow, debounce and control reset

### SUMMARY
Add some polish to the new Handlebars plugin:
- add debounce to template and style controls to avoid rerendering on each keypress
- fix height and width to fix buggy overflow + default to auto scroll
- add `stringify` helper to make it more convenient to inspect the rows in the data array
- update example template and CSS code to be more intuitive
- avoid resetting controls when switching between aggregate and raw mode
- clean up dependencies
- remove redundant comments/code

### AFTER

https://user-images.githubusercontent.com/33317356/165733042-75f52dce-b65f-491a-a5f3-bd80b5006460.mp4

### BEFORE

https://user-images.githubusercontent.com/33317356/165733132-84111740-4d3c-48bd-b7e1-2ba96735363b.mp4

### TESTING INSTRUCTIONS
1. Create a chart with the handlebars plugin
2. add a few columns to aggregate and raw mode and notice they are no longer reset when switching between query modes
3. run a query that causes the chart to overflow and see it scroll cleanly
4. edit the template/CSS code and notice the dbounce

### ADDITIONAL INFORMATION
